### ISSUE TITLE
Issue with name/tensor (fake_A and fake_B)

### ISSUE THREAD

**Issue by 2Falx** [cchen-cc/SIFA#53]
Hello! :)
I am working on a project on image to image translation with cerebral images and I'm using SIFA.
I have a doubt about the way you save the images and I hope you can help me.
In main.py. line 288 ( "for name, tensor in zip(names, tensors)" ) you iterate and save all the 6 images that SIFA produces.
Given lines 258 and 285: 
           names =  [      'inputA_', '              inputB_',          **'fakeA_',            'fakeB_'**,            'cycA_',        'cycB_']
           tensors = [inputs['images_i'], inputs['images_j'], **fake_B_temp, fake_A_temp**, cyc_A_temp, cyc_B_temp]
The thing I don't understand is why you call the tensor fake_B_temp with the name fake A and viceversa.
Is this fake_B_temp the image transformed from source into target or is the image from target into source?
Thanks in advance.

**Comment by chke097** [comment]
@2Falx 
May I ask whether your training model is successful? My training model has this error:

caused by op 'shuffle_batch', defined at:
File "main.py", line 626, in
main(config_filename='./config_param.json')
File "main.py", line 622, in main
sifa_model.train()
File "main.py", line 355, in train
self.inputs = data_loader.load_data(self._source_train_pth, self._target_train_pth, True)
File "/data2/xxb/SIFA-master/SIFA-master/data_loader.py", line 112, in load_data
1,50,10,num_threads=4)
File "/data2/xxb/anaconda/envs/tensorflow10/lib/python3.6/site-packages/tensorflow/python/training/input.py", line 1300, in shuffle_batch
name=name)
File "/data2/xxb/anaconda/envs/tensorflow10/lib/python3.6/site-packages/tensorflow/python/training/input.py", line 846, in _shuffle_batch
dequeued = queue.dequeue_many(batch_size, name=name)
File "/data2/xxb/anaconda/envs/tensorflow10/lib/python3.6/site-packages/tensorflow/python/ops/data_flow_ops.py", line 476, in dequeue_many
self._queue_ref, n=n, component_types=self._dtypes, name=name)
File "/data2/xxb/anaconda/envs/tensorflow10/lib/python3.6/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 3480, in queue_dequeue_many_v2
component_types=component_types, timeout_ms=timeout_ms, name=name)
File "/data2/xxb/anaconda/envs/tensorflow10/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
op_def=op_def)
File "/data2/xxb/anaconda/envs/tensorflow10/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py", line 454, in new_func
return func(*args, **kwargs)
File "/data2/xxb/anaconda/envs/tensorflow10/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3155, in create_op
op_def=op_def)
File "/data2/xxb/anaconda/envs/tensorflow10/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1717, in init
self._traceback = tf_stack.extract_stack()

OutOfRangeError (see above for traceback): RandomShuffleQueue '_2_shuffle_batch/random_shuffle_queue' is closed and has insufficient elements (requested 1, current size 0)
[[Node: shuffle_batch = QueueDequeueManyV2[component_types=[DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT], timeout_ms=-1, _device="/job:localhost/replica:0/task:0/device:CPU:0"](shuffle_batch/random_shuffle_queue, shuffle_batch/n)]]

How to solve this problem？

**Comment by cchen-cc** [comment]
@2Falx fake_B_temp is the image transformed from source into target. Since the image save part of the code is adapted from CycleGAN implementation, some naming is inconsistent with my model. As long as you are clear about the meaning of the outputs from the model, the naming can be adjusted based on your preference.
